src/services/localStorageService.js

```diff
@@ -0,0 +1,25 @@
+export const addToHistory = (placeName, placeId, lat, lng) => {
```

Comment:
[optional]
I think when we have more than 3 arguments in a function it worth to pass them throw the object. It guarantees us we don't miss order.